x-pack/plugins/data_enhanced/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "./target/types",
+    "emitDeclarationOnly": true,
+    "declaration": true,
+    "declarationMap": true,
+  },
+  "include": [
+    "common/**/*",
+    "public/**/*",
+    "public/autocomplete/providers/kql_query_suggestion/__fixtures__/*.json",
```

Comment:
Why is `"public/autocomplete/providers/kql_query_suggestion/__fixtures__/*.json"` explicitly needed for json files? I don't see any similar inclusion in 

https://github.com/elastic/kibana/blob/1ec2f1d0784c39a9c34f1fe018077d26f06a8237/x-pack/tsconfig.json#L3

Reply:
It seems to be a bug in `tsc` when building composite project with `resolveJsonModule: true` https://github.com/microsoft/TypeScript/issues/25636
I will add a comment to the `tsconfig.json` file.